### PR TITLE
Some text alterations

### PR DESCRIPTION
**Pull request by slash3b**

### CODE REVIEW

File: docs/days/day-6.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ In the MVC model, the Model defines all the business logic, and the Controller o
 As the code returns a collection of jobs, let’s move the code to the repository, that is part of model layer.
 For that we will need to create a custom `repository` class for Job entity and to add the query to that class.
 
+At this point, you may have a question - what is this Repository class anyway? 
+Repository is a pattern, a common solution to the well know problem. It has been around for quite some time and was popularizer around 90's by people like Martin Forwler and Eric Evans. If you try to describe it briefly it can get quite complex. For now think of it as just another layer of abstraction
```
Comment (gregurco) [inline docs/days/day-6.md]: Fowler, not Forwler

Comment (gregurco) [reply docs/days/day-6.md]: popularized, not popularizer

File: docs/days/day-6.md [hunk 2]
```diff
@@ -127,6 +127,10 @@ In the MVC model, the Model defines all the business logic, and the Controller o
...
+above the Entity that contains all the useful methods to work with database. If you are still curious about Repository pattern continue to learn [here][4]
```
Comment (gregurco) [inline docs/days/day-6.md]: put a `.` at the end of sentence

File: docs/days/day-6.md [hunk 3]
```diff
@@ -151,7 +155,7 @@ class JobRepository extends EntityRepository
 ```
 
 Next, add a new method, `findActiveJobs()`, to the newly created repository class.
-This method will query for all of the active Job entities sorted by the expiresAt column (and filtered by category if it receives the $categoryId parameter).
+This method will query for all of the active Job entities sorted by the `expiresAt` column and filtered by category if it receives the `$categoryId` parameter. The method will return special ArrayCollection object from Doctrine bundle, that object will containt all results and can be iterated with foreach just like the usual array.
```
Comment (gregurco) [inline docs/days/day-6.md]: contain, not containt

**Review by gregurco — CHANGES_REQUESTED** [review]
:+1: good remarks. I wrote some small comments.

**Comment by slash3b** [comment]
Thanks! I will do corrections tonight, hopefully I wont be sleepy that much :D

File: docs/days/day-6.md (hunk 1 above)
Comment (slash3b) [reply docs/days/day-6.md]: one of these days I will get enough sleep

**Comment by gregurco** [comment]
@slash3b :+1: thank you!

**Comment by slash3b** [comment]
^^

**Pull request merged**